extensions/wikia/Recirculation/js/helpers/DiscussionsHelper.js

```diff
@@ -6,44 +6,32 @@ define('ext.wikia.recirculation.helpers.discussions', [
 ], function ($, w, abTest, nirvana) {
 	'use strict';
 
-	return function(config) {
-		var defaults = {
-				type: 'all',
-				cityId: w.wgCityId
+	var deferred = $.Deferred();
+
+	function prepare() {
+		return deferred.promise();
+	}
+
+	function fetch() {
+		nirvana.sendRequest({
+			controller: 'Recirculation',
+			method: 'discussions',
+			format: 'html',
+			type: 'get',
+			data: {
+				cityId: w.wgCityId,
+				latest: true,
+				limit: 10,
+				ignoreWgEnableRecirculationDiscussions: true
```

Comment:
what is the purpose of ignoring `ignoreWgEnableRecirculationDiscussions`?

Reply:
this is temporary until we will have MCF enabled everywhere